docs/workspaces.md

```diff
@@ -60,6 +60,20 @@ they have independent behaviour and no cluster workspace sees objects from other
 cluster workspaces. In contrast to namespace in Kubernetes, this includes non-namespaced
 objects, e.g. like CRDs where each workspace can have its own set of CRDs installed.
 
+## User Home Workspaces
+
+User home workspaces are an optional feature of kcp. If enabled (through `--enable-home-workspaces`), there is a special virtual `Workspace` called `~` in the root workspace. It is used by `kubectl ws` (which similarly to Unix `cd` move the users to their home) to derive the full path to the user home workspace. This full path depends on the user name, the configured number (default 2), the length (default 2) of bucket workspaces and the root prefix (default `root:users`). With that a user `adam` is mapped to e.g. `root:users:a8:f1:adam`. The bucket names are deterministically derived from the user name (via some hash).
+
+The user home workspace, if enabled, is logically pre-existing for every user, i.e. the user is given that impression. In reality, the home workspace is created on-demand on access, but this creation is not visible from outside.
```

Comment:
Who does this implementation detail impact? Lets' entirely remove the line.

Reply:
it impacts admins and ppl trying to understand the system. They docs are not user-docs, not even admin docs.

Reply:
Hm, docs without a target reader are kind of a smell. If we keep the line, @pweil- can we please edit it to something like:

> Home user workspaces are created on-demand when they are first accessed, but this is not visible to the user, allowing the system to only incur the cost of these workspaces when they are needed.

Reply:
done